server/lib/utils.js

```diff
@@ -412,17 +412,19 @@ export function exportToPDF(template, data, options) {
 
 /**
  * Default host id, set this for new collectives created through our flow
+ *
+ * @param {"opensource" | null} category of the collective
  */
-export const defaultHostCollective = tag => {
+export const defaultHostCollective = category => {
   if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging') {
-    if (tag === 'opensource') {
```

Comment:
not "open source"?

Reply:
I was hesitating on renaming these. We call this function from multiple places and we never really pass any tags to it, only static strings. So I preferred renaming the parameter from `tag` to `category` to remove any confusion